discovery_child_development/utils/bigquery.py

```diff
@@ -18,6 +18,27 @@
 dotenv.load_dotenv()
 
 
+def find_credentials(credentials_env_var: str):
```

Comment:
Does it return dict?

```suggestion
def find_credentials(credentials_env_var: str) -> dict:
```

Reply:
It returns a PosixPath so I will indicate that accordingly